clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -467,78 +492,6 @@ static Error getSYCLDeviceLibs(SmallVector<std::string, 16> &DeviceLibFiles,
   return Error::success();
 }
 
-static bool isStaticArchiveFile(const StringRef Filename) {
```

Comment:
All recent changes related to unbundling of device objects have been removed. Once we turn on the new offloading model by default, all library developers are expected to recompile their files using the new compiler. This proposal will be discussed with the 'SYCL upstreaming' team and we will add backward compatibility if required in a later PR.